### DISCUSSION TITLE
Wired style

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#347] [Q&A] [answered]
Hello, I would like to draw a direct wiring like the one in the attached image, how can I do it?
![RDT_wired_style](https://github.com/ra3xdh/qucs_s/assets/70080894/ab82e2ed-ccae-42a9-9ecc-3153bd7e00a3)

**Comment by ra3xdh** [comment, marked as answer]
There is no way to draw straight wires on schematic. The schematic editor supports only 90 degree aligned wires.
